protein-engineering-design-and-selection.csl

```diff
@@ -16,7 +16,8 @@
     <issn>1741-0126</issn>
     <eissn>1741-0134</eissn>
     <published>2014-09-06T21:00:00-06:00</published>
-    <updated>2014-09-07T00:00:00-06:00</updated>
+    <updated>2014-09-11T02:57:16+00:00</updated>
+    <updated>2016-09-20T22:05:00-04:00</updated>
```

Comment:
you have two update timestamps. I think that's the only issue that prevent validation from goting through.